src/cloud/auth.rs

```diff
@@ -70,12 +79,157 @@ pub async fn do_login(client: &CloudClient) -> anyhow::Result<()> {
     )
 }
 
-pub async fn logout(_c: &options::Logout) -> anyhow::Result<()> {
-    write_json(
-        &cloud_config_file()?,
-        "cloud config",
-        &CloudConfig { access_token: None },
-    )?;
-    print::success("You're now logged out from EdgeDB Cloud.");
-    Ok(())
+fn find_real_project_dirs(
+    f: impl Fn(&str) -> bool,
+) -> anyhow::Result<HashMap<String, Vec<PathBuf>>> {
+    find_project_dirs("cloud-profile", f, false).map(|projects| {
```

Comment:
```suggestion
    let projects = find_project_dirs("cloud-profile", f, false)?;
```
No need for map and nesting here.